node/src/reactor.rs

```diff
@@ -679,13 +721,15 @@ where
             match TERMINATION_REQUESTED.load(Ordering::SeqCst) as i32 {
                 0 => {
                     if let Some(reactor_exit) = self.reactor.maybe_exit() {
-                        return reactor_exit;
+                        break reactor_exit;
+                    }
+                    if !self.crank(rng).await {
+                        break ReactorExit::ProcessShouldExit(ExitCode::Abort);
                     }
-                    self.crank(rng).await;
                 }
-                SIGINT => return ReactorExit::ProcessShouldExit(ExitCode::SigInt),
-                SIGQUIT => return ReactorExit::ProcessShouldExit(ExitCode::SigQuit),
-                SIGTERM => return ReactorExit::ProcessShouldExit(ExitCode::SigTerm),
+                SIGINT => break ReactorExit::ProcessShouldExit(ExitCode::SigInt),
+                SIGQUIT => break ReactorExit::ProcessShouldExit(ExitCode::SigQuit),
+                SIGTERM => break ReactorExit::ProcessShouldExit(ExitCode::SigTerm),
```

Comment:
Using `break` instead of `return` here doesn't change anything, does it?

Reply:
No, I just wanted to be consistent and I think `break` is a little bit cleaner.